### ISSUE TITLE
Type errors when transpiling

### ISSUE THREAD

**Issue by evanjo03** [joeattardi/emoji-button#182]
There seem to be a few typescript issues around the usage of twemoji.  Additionally, the @types/twemoji dependency is for a different /incompatible major version of twemoji.

```
node_modules/@joeattardi/emoji-button/dist/types.d.ts:2:10 - error TS2614: Module '"twemoji"' has no exported member 'ParseObject'. Did you mean to use 'import ParseObject from "twemoji"' instead?

import { ParseObject } from 'twemoji';

node_modules/twemoji/index.d.ts:32:1 - error TS1046: Top-level declarations in .d.ts files must start with either a 'declare' or 'export' modifier.
```

**To Reproduce**
1. `npm init @open-wc` and follow prompts
2. `cd [project-name] && npm i`
3. `npm i @joeattardi/emoji-button`
4. Add `import { EmojiButton } from "@joeattardi/emoji-button";`
5. `npm run tsc`
6. See error

**Expected behavior**
Tsc occurs without issues.

**Screenshots**
![image](https://user-images.githubusercontent.com/42701217/158400054-91f8f89b-1180-4575-8654-94091dace4d7.png)

**Environment**
Chrome 99.0.4844.51

**Comment by Saul-Mirone** [comment]
Seem here. The second error has been fixed by twemoji, but there're still some error on the `emoji-button` side.

**Comment by Saul-Mirone** [comment]
Seems replace [this line](https://github.com/joeattardi/emoji-button/blob/d859569bb3d9d9180c2479699396e0ec5bc7b899/src/types.ts#L2) with `import { TwemojiOptions } from 'twemoji'` can solve this issue.

**Comment by joeattardi** [comment]
Thanks for spotting this! I think I may have inadvertently upgraded twemoji and not the types. Twemoji 13.x has its own types included but doesn't export them all so it's a bit of a headache. 

I am getting close to a first cut of the v5 release, a major rewrite. This version changes how Twemoji is used so it shouldn't be an issue. 

In the meantime I created a new release of 4.x that should resolve this for now by downgrading Twemoji back to 12.x to match the typings. I haven't had a chance to publish to npm yet but I will be doing that later this evening.  

I used your minimal test case (thank you for providing that!) and the TS compilation is successful now.
